build.gradle.kts

```diff
@@ -34,12 +93,36 @@ kotlin {
                 implementation(dependencies.platform("io.kotest:kotest-bom:5.9.1"))
                 runtimeOnly("io.kotest:kotest-runner-junit5")
                 implementation("io.kotest:kotest-framework-api")
+                implementation("io.kotest:kotest-framework-datatest")
                 implementation("io.kotest:kotest-assertions-core")
+
+                implementation("it.krzeminski:snakeyaml-engine-kmp:3.1.0")
+                implementation("io.github.optimumcode:json-schema-validator:0.3.1")
+            }
+        }
+
+        jsTest {
+            dependencies {
+                implementation(dependencies.platform("io.kotest:kotest-bom:5.9.1"))
+                implementation("io.kotest:kotest-framework-engine")
+                implementation("io.kotest:kotest-framework-api")
+                implementation("io.kotest:kotest-framework-datatest")
+                implementation("io.kotest:kotest-assertions-core")
```

Comment:
Can we move at least some of these to `commonTest`?

Reply:
We surely could, it is just bad practice from Gradle point of view.
Each source set should always declare directly which dependencies it is using directly and never use transitive dependencies.

I mean, it is your project, if you want it that way I can change it of course and probably as long as you do not add the [dependency-analysis-gradle-plugin](https://github.com/autonomousapps/dependency-analysis-gradle-plugin) which should hopefully complain about that. :-)

Reply:
Hmm, I get it. If we used these from the common source set, it would be fine, but we happen to use them from different platform-specific tests. Ok, let's leave it as is...